pkgs/tools/misc/gaphor/default.nix

```diff
@@ -0,0 +1,62 @@
+{ lib
+, python3
+, fetchurl
+, fetchFromGitHub
+, gobject-introspection
+, gtksourceview4
+, wrapGAppsHook
+, makeDesktopItem
+, copyDesktopItems
+}:
+
+let
+  app_icon = fetchurl {
+    url = "https://gaphor.org/images/gaphor.svg";
+    sha256 = "sha256-xLvkpnZi1SwEy9KDsz8/86l2l+FYxWxLd2wcTvUn3WI=";
+  };
+
+in python3.pkgs.buildPythonApplication rec {
+  pname = "gaphor";
+  version = "2.6.5";
+
+  format = "pyproject";
+
+  src = python3.pkgs.fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-IFsbWx5lblKsnEibVihM6ZPRoydXC+JM1gdZEUUTKxw=";
+  };
+
+  strictDeps = false;
```

Comment:
Ready, I have specified the issue with the `strictDeps`

Reply:
Have you looked into this issue in more detail? It [does seem](https://github.com/gaphor/gaphor/commit/77aaf37846625f8e5306ae53ae9c866a3d08451d) like PangoCairo is quite important for the package's functionality. Maybe [this issue](https://github.com/qtile/qtile/issues/1473) helps.

Reply:
So I've found #56943 which recommends setting `gobject-introspection` on both `nativeBuildInputs` and `buildInputs`. Honestly, I feel this is too hacky and ugly (specifically because it will need a comment specifying why this is applied), but as this avoids using the `strictDeps` and considers the usage of `pango`, then I will just skip my attention from that until the issue gets solved.